### PR TITLE
Documentation root path

### PR DESCRIPTION
**Pull request by sienna-kopf**
# SOM-Timer BE Pull Request Form

## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
- Add documentation page to the root path
- Build page with HTML and style with CSS
- Use FE colors and fonts to show resemblance 

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Has This Been Tested?
- [ ] No
- [x] Yes

- Attempt to write tests for the html page. Currently only checking for certain text attributes. Trying to find something like Capybara in Rails to aid with this. 
- all tests passing

# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs/pry sessions
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: tests/test_documentation.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        self.assertEquals(response.status, "200 OK")
+
+        html = response.data.decode()
+        assert "Som Timer Endpoints" in html
+        assert "GET  '/api/timers/{timer_id}'" in html
+        assert "PUT  '/api/timers/{timer_id}'" in html
+        assert "GET  '/api/rand_exercise'" in html
+
+if __name__ == "__main__":
+    unittest.main()
```
Comment (Chulstro) [inline tests/test_documentation.py:35]: Is there a way to test this in a way that we tested our view pages in Rails? I just wonder if we can test the display as well

**Review by Chulstro — APPROVED** [review]
It looks really good! I like the use of an html file that is read by an open function

File: tests/test_documentation.py (hunk 1 above)
Comment (sienna-kopf) [reply tests/test_documentation.py:35]: Right!! I did a  little digging and didn't find anything too good. If we make the change to Mamba, that might help us.

File: application/templates/documentation.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>SomTimer</title>
```
Comment (rwilliams659) [inline application/templates/documentation.html:4]: This is so awesome, can't wait to see what it looks like

**Review by rwilliams659 — APPROVED** [review]
This is great! Nice job BEEs 🥇

File: application/static/styles/mainpage.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@400;600&display=swap');
+
+html {
+  background: linear-gradient(45deg, #7987a1, #C9D2DF);
```
Comment (Abdeboskey) [inline application/static/styles/mainpage.css:4]: YES linear gradient! I popped a linear gradient on the FE after we did this!

File: application/templates/documentation.html (hunk 2 above)
Comment (Abdeboskey) [reply application/templates/documentation.html:4]: Same, this is such a rad addition to the BE.

File: tests/test_documentation.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+        response = self.test_app.get(
+            '/',
+            follow_redirects=True
+        )
+
+        self.assertEquals(response.status, "200 OK")
+
+        html = response.data.decode()
+        assert "Som Timer Endpoints" in html
```
Comment (Abdeboskey) [inline tests/test_documentation.py:29]: Whoa this is interesting testing!!!

**Review by Abdeboskey — APPROVED** [review]
Nice job. It feels really nice to just see some good ol' HTML and css on our Mod 4 capstone project. 🤘🏼

**Pull request merged**